cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js

```diff
@@ -235,10 +238,10 @@ const renderModelDetails = (model, newlyTrainingModel, experimentId) => {
           <div>{model.description || '--'}</div>
         </div>
         <div>
-          <strong># Directives </strong>
+          <strong># Directives ({directivesCount}) </strong>
```

Comment:
from the mock it seems like we don't need the '#' anymore?

Reply:
👍 Removed `#`.